### ISSUE TITLE
Ctrl+E does not stop editing

### ISSUE THREAD

**Issue by abreffni** [notable/notable#1191]
- **OS Version**: win32 10.0.18363
- **Notable Version**: v1.9.0-alpha.4

### Ctrl+E (^E) should (based on pre-alpha builds) allow you to toggle back and forth between edit and preview modes.

**Comment by fabiospampinato** [comment]
> Ctrl+E (^E) should (based on pre-alpha builds) allow you to toggle back and forth between edit and preview modes.

Yes but actually the issue here is that inside an editor Ctrl+E is also bound to another shortcut
